src/test/java/it/okkam/validation/FiscalCodeValidatorTest.java

```diff
@@ -54,8 +55,10 @@ public void testComuniMapInitialization() throws IOException {
   @Before
   public void init() throws IOException {
     String codiciIstatStr = readLocalFile(CODICE_ISTAT_COMUNI_CSV);
-    conf1 = FiscalCodeValidator.getFiscalCodeConf(codiciIstatStr, maxComuneNameLength, "M", 8, 10, 3, 5, 0, 2);
-    conf2 = FiscalCodeValidator.getFiscalCodeConf(codiciIstatStr, maxComuneNameLength, "M", 2, 4, 5, 7, 8, 10);
+    conf1 = FiscalCodeValidator.getFiscalCodeConf(codiciIstatStr, maxComuneNameLength, "M", 8, 10,
+        3, 5, 0, 2);
+    conf2 = FiscalCodeValidator.getFiscalCodeConf(codiciIstatStr, maxComuneNameLength, "M", 2, 4, 5,
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Make the enclosing method "static" or remove this set. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://dev.okkam.it/sonar/coding_rules#rule_key=squid%3AS2696)